docs/samples/ticket.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const RippleAPI = require('../../src').RippleAPI; // require('ripple-lib')
+
+const address = 'INSERT ADDRESS HERE';
+const secret = 'INSERT SECRET HERE';
+
+const api = new RippleAPI({server: 'wss://s1.ripple.com:443'});
+const instructions = {
+  maxLedgerVersionOffset: 5
+};
+const numberOfTickets = 1;
+
+function quit(message) {
+  console.log(message);
+  process.exit(0);
+}
+
+function fail(message) {
+  console.error(message);
+  process.exit(1);
+}
+
+api.connect().then(() => {
+  console.log('Connected...');
+  return api.prepareTicket(address, numberOfTickets, instructions).then(prepared => {
```

Comment:
I think it may be more consistent to rename this to `prepareTicketCreate` because `TicketCreate` is the transaction type